biggraphite/drivers/_utils.py

```diff
@@ -18,6 +18,8 @@
 
 import threading
 
+from opencensus.trace import execution_context
```

Comment:
is there a way to make it optional if the lib is not installed ?

Reply:
Done